rs/nervous_system/candid_utils/src/printing.rs

```diff
@@ -0,0 +1,10 @@
+use candid::{CandidType, IDLValue};
+
+pub fn pretty<T: CandidType>(value: T) -> Result<String, String> {
```

Comment:
Should this really consume? Esp since try_from_candid_type takes a reference, it really does not seem like caller should lose ownership of `value`.